vyxal/main.py

```diff
@@ -49,7 +49,7 @@
     Ṫ    Print the sum of the entire stack
     ṡ    Print the entire stack, joined on spaces
     J    Print the entire stack, separated by newlines.
-    t    Lists are considered truthy if they are not empty
+    t    Vectorise boolify on Lists
```

Comment:
Hang on, isn't it the case that now all that this PR does is invert the meaning of `t`?

Reply:
:face_palm: er yes

Reply:
How should I change it?